esmvalcore/preprocessor/_regrid.py

```diff
@@ -332,11 +372,12 @@ def regrid(cube, target_grid, scheme, lat_offset=True, lon_offset=True):
 
     # Perform the horizontal regridding.
     if _attempt_irregular_regridding(cube, scheme):
-        cube = esmpy_regrid(cube, target_grid, scheme)
+        regridded_cube = esmpy_regrid(cube, target_grid, scheme)
     else:
-        cube = cube.regrid(target_grid, HORIZONTAL_SCHEMES[scheme])
+        regridded_cube = cube.regrid(target_grid, HORIZONTAL_SCHEMES[scheme])
+        _add_regridded_aux_coords(cube, regridded_cube, target_grid)
```

Comment:
ah okay so a check on the existence of the aux coord is rather needed then

Reply:
See comment above, l. 197